### ISSUE TITLE
Allow a dataset to be updated

### ISSUE THREAD

**Issue by xleclercq** [wxIshiko/wxCharts#30]
Adding a single value to a dataset of a line chart for instance should be possible without replacing the entire dataset.

**Comment by Anton-V-K** [comment]
Any plans to implement this feature? :)
I guess, many (most? all?) charts doesn't support updating the underlying dataset (I wasn't able to find the relevant methods in `wxCandlestickChart`). Without such functionality the library is hardly usable for dynamic data.

**Comment by Kvaz1r** [comment]
> I guess, many (most? all?) charts doesn't support updating the underlying dataset (I wasn't able to find the relevant methods in `wxCandlestickChart`). Without such functionality the library is hardly usable for dynamic data.

Yes, because it's really big issue. But if this or any other feature more needed in one special chart it would be better to open a separate issue. Much easier provide solution for one chart than for all of them.
